ToolsQA/Tests/Driver.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Firefox;
+using ToolsQA.Pages;
+
+namespace ToolsQA.Selenium_Basics
+{
+    public static class Driver
+    {
+        private static IWebDriver _driver;
+
+        public static IWebDriver DriverInstance{ get => _driver; }
+
+        public static void StartBrowser()
+        {
+            Driver._driver = new FirefoxDriver();
+        }
+
+        public static void OpenPage(string url)
+        {
+            Driver._driver.Url = url;
+        }
+
+        public static void StopBrowser()
```

Comment:
CloseBrowser would be a better name - it would reflect what you're doing in the method body